### PR TITLE
Generate flip python gprc files in package.

### PR DESCRIPTION
**Pull request by sanebay**
Based on brian and hari's suggestions use flip python client in nublox verify.

### CODE REVIEW

File: src/flip/proto/CMakeLists.txt [hunk 1]
```diff
@@ -16,3 +16,18 @@ target_link_libraries(flip_proto
       protobuf::libprotobuf
       gRPC::grpc++
   )
+
+set(FLIP_PROTO_PY_GEN flip_server_pb2_grpc.py
+    flip_server_pb2.py
+    flip_spec_pb2_grpc.py
+    flip_spec_pb2.py)
+
+add_custom_command(OUTPUT ${FLIP_PROTO_PY_GEN}
```
[the remark this answers is not in the document]
Comment (sanebay) [reply src/flip/proto/CMakeLists.txt:25]: I tried this but its only generating protobuf files and not the grpc files.
root@sdsbuild07:/tmp/source/sisl/build_flip_python_v8x# make -j8 flip_py
[ 50%] Running python protocol buffer compiler on flip_server.proto
[100%] Running python protocol buffer compiler on flip_spec.proto
[100%] Built target flip_py
root@sdsbuild07:/tmp/source/sisl/build_flip_python_v8x# find . -name "*.py"
./src/flip/proto/flip_server_pb2.py
./src/flip/proto/flip_spec_pb2.py
root@sdsbuild07:/tmp/source/sisl/build_flip_python_v8x#

File: src/flip/proto/CMakeLists.txt [hunk 2]
```diff
@@ -16,3 +16,18 @@ target_link_libraries(flip_proto
       protobuf::libprotobuf
       gRPC::grpc++
   )
+
+set(FLIP_PROTO_PY_GEN flip_server_pb2_grpc.py
```
[the remark this answers is not in the document]
Comment (sanebay) [reply src/flip/proto/CMakeLists.txt]: Removed it

File: src/flip/proto/CMakeLists.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (sanebay) [reply src/flip/proto/CMakeLists.txt]: Ack.

**Review by szmyd — APPROVED** [review]
lgtm

**Pull request merged**